src/Tests/TMGMTCKEditorHtmlTagTest.php

```diff
@@ -60,6 +60,24 @@ public function testReviewForm() {
     $node = $this->drupalGetNodeByTitle('Segments test');
     $this->drupalGet('node/' . $node->id());
 
+    // Node text values.
+    $title = 'Segments test';
+
+    $masked_body = '<tmgmt-segment id="1">This is the first segment.</tmgmt-segment><tmgmt-tag element="br /" raw="&lt;br /&gt;" />';
```

Comment:
Should we mask only the tags that are inside the `<tmgmt-segment>` tags or also `<br />` and others that are outside?